menas/src/main/scala/za/co/absa/enceladus/menas/LineageConfig.scala

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2018-2019 ABSA Group Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package za.co.absa.enceladus.menas
+
+import za.co.absa.enceladus.utils.general.ConfigReader
+
+object LineageConfig {
+  final val apiUrl: Option[String] = ConfigReader.readStringConfigIfExist("menas.lineage.readApiUrl")
+  final val jarName =  "spline"
+  final val subSpace = "lineage"
+
+  final val mappingPathForController = "/" + subSpace + "/app/**"
+  final val resourceHandler = "/" + subSpace + "/**"
+  final val resourceLocation = "/webjars/" + jarName + "/"
+  final val executionIdApiTemplate: Option[String] = apiUrl.map(_ + "/execution-events?dataSourceUri=%s&applicationId=%s")
```

Comment:
Could use string interpolation

Reply:
You mean in the `map`? There I considered the need of having a variable made it more complex.
In the ones above - if string interpolation is used, the `val` is not considered a constant, and cannot be used in annotations then. `+` is allowed though.

Reply:
I see, curious